packages/manager/modules/netapp/src/dashboard/template.html

```diff
@@ -97,6 +97,7 @@
                 <ovh-advices
                     service-name="{{ ::$ctrl.storage.id }}"
                     service-type="netapp"
+                    on-advice-click="$ctrl.onAdviceClick(advice)"
```

Comment:
Not using the parameter` (advice) `in the function ?

Reply:
I let it to highlight that we can know which advice is passed.
But not used from controller side.